spec/classes/foreman_proxy__plugin__abrt_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe 'foreman_proxy::plugin::abrt' do
+
+  let :facts do {
+    :osfamily               => 'RedHat',
+    :operatingsystem        => 'CentOS',
+    :operatingsystemrelease => '6.5',
+    :fqdn                   => 'my.host.example.com',
+  } end
+
+  describe 'with default settings' do
+    let :pre_condition do
+      "include foreman_proxy"
+    end
+
+    it { should contain_foreman_proxy__plugin('abrt') }
+    it 'should configure abrt.yml' do
+      should contain_file('/etc/foreman-proxy/settings.d/abrt.yml').
```

Comment:
I think the following should suffice. Not sure if undef resolves to nil so that may not work:

```puppet
should contain_foreman_proxy__settings_file('abrt').
  with_template_path('foreman_proxy/plugin/abrt.yml.erb').
  with_group(nil)
```